### ISSUE TITLE
Notary v2: RC1

### ISSUE THREAD

**Issue by SteveLasker** [notaryproject/specifications#70]
Notary V2 RC1 will be a time & quality-based release targeting the Fall of 2021.

Notary v2 RC1 will be dependent upon [Artifacts RC1 (Reference Type Support)](#68)

## Scope

The scope of RC1 is targeted to be complete by June 2021, providing time for a summer 2021 release and fall dependency for consumers.

**Comment by mnm678** [comment]
We need to update this milestone. The Notary project requirements and threat model are still under active development, so I don't think it's possible to have a release of an implementation of these requirements before the end of the year at the earliest.

**Comment by dtzar** [comment]
@iamsamirzon @SteveLasker @FeynmanZhou @yizha1 
I propose we close roadmap issues like this one and these below since the roadmap repo should be covering this.  This notaryproject repo can be for proposed high-level spec/designs/scenarios before they are a user story notation* implements.

1. https://github.com/notaryproject/notaryproject/issues/52
2. https://github.com/notaryproject/notaryproject/issues/68
3. https://github.com/notaryproject/notaryproject/issues/69

**Comment by FeynmanZhou** [comment]
> @iamsamirzon @SteveLasker @FeynmanZhou @yizha1 I propose we close roadmap issues like this one and these below since the roadmap repo should be covering this. This notaryproject repo can be for proposed high-level spec/designs/scenarios before they are a user story notation* implements.
> 
> 1. [Prototype 3 Scope DRAFT #52](https://github.com/notaryproject/notaryproject/issues/52)
> 2. [Artifacts: RC1 (Reference Type Support) #68](https://github.com/notaryproject/notaryproject/issues/68)
> 3. [Artifacts: RC2 (Versioned Artifact Support) #69](https://github.com/notaryproject/notaryproject/issues/69)

@dtzar Why not transfer these issues to the Roadmap repo? I think they are still under discussion.

**Comment by iamsamirzon** [comment]
> > @iamsamirzon @SteveLasker @FeynmanZhou @yizha1 I propose we close roadmap issues like this one and these below since the roadmap repo should be covering this. This notaryproject repo can be for proposed high-level spec/designs/scenarios before they are a user story notation* implements.
> > 
> > 1. [Prototype 3 Scope DRAFT #52](https://github.com/notaryproject/notaryproject/issues/52)
> > 2. [Artifacts: RC1 (Reference Type Support) #68](https://github.com/notaryproject/notaryproject/issues/68)
> > 3. [Artifacts: RC2 (Versioned Artifact Support) #69](https://github.com/notaryproject/notaryproject/issues/69)
> 
> @dtzar Why not transfer these issues to the Roadmap repo? I think they are still under discussion.

@dtzar  - I am aligned with you to close these items highlighted above . But before we move them as is in the roadmap repo, lets see if these are still needed as is, or should we close them and create net new in roadmap repo with the desired feature. I believe we have already implemented these items.

**Comment by dtzar** [comment]
I agree there is some value in extracting the missing ideas/features from the 1-3 items into the roadmap repo.  As you mentioned Samir, some of the things have been implemented or may not make sense at this time.

This particular issue we're commenting in I don't see any merit in.

I also don't like the idea of any "RC" milestone or dates being listed in the notaryproject repo.  This repo should just be for future ideas/designs - _pre-backlog_.  

We could also consider modifying the 1-3 issues there into more pure spec/design ideas (i.e. remove the RC/timelines and such) which still sense, then adding new user story(ies) on the roadmap repo and link to those.

**Comment by FeynmanZhou** [comment]
Close this issue as Notation v1.3.0 is now the latest release.
